sql/migrations/warehouse/000025_drop_trigger_schema_versions.sql.up.sql

```diff
@@ -0,0 +1,2 @@
+
+DROP TRIGGER IF EXISTS warehouse_schema_history_trigger ON wh_schemas;
```

Comment:
Should we also get rid of the `wh_schema_versions` table as well?

Reply:
I am cautious when deleting data using migrations. `wh_schema_versions` should be a safe table to delete, my only concern are hidden dependencies from open source customers.